usr/share/rear/output/PXE/default/800_copy_to_tftp.sh

```diff
@@ -32,14 +32,17 @@ else
     [[ ! -d "$PXE_TFTP_LOCAL_PATH" ]] && mkdir $v -m 750 "$PXE_TFTP_LOCAL_PATH" >&2
 fi
 
-
-cp -pL $v "$KERNEL_FILE" "$PXE_TFTP_LOCAL_PATH/$PXE_KERNEL" || Error "Failed to copy KERNEL_FILE '$KERNEL_FILE'"
-cp -a $v "$TMP_DIR/$REAR_INITRD_FILENAME" "$PXE_TFTP_LOCAL_PATH/$PXE_INITRD" || Error "Failed to copy initrd '$REAR_INITRD_FILENAME'"
+# Follow symbolic links to ensure the real content gets copied
+# but do not preserve mode,ownership,timestamps (i.e. no -p option) because that may fail (on sshfs) like
```

Comment:
I vaguely remember I made that comment somewhere...
Yes, it was https://github.com/rear/rear/pull/2684
because of https://github.com/rear/rear/issues/2683